### PR TITLE
Nuke Ops and Traitor Update+Balance

### PR DESCRIPTION
**Pull request by Fox-McCloud**
A port of:
- https://github.com/tgstation/-tg-station/pull/9800
- https://github.com/tgstation/-tg-station/pull/9510
- https://github.com/tgstation/-tg-station/pull/10216
- https://github.com/tgstation/-tg-station/pull/10765
- https://github.com/tgstation/-tg-station/pull/11207
- https://github.com/tgstation/-tg-station/pull/11146
- https://github.com/tgstation/-tg-station/pull/11018
- https://github.com/tgstation/-tg-station/pull/11497

Things I didn't port: storage implant (we have compressed matter) and the syndicate MMI (this will require further changes; will look into adding it later).

Basically a pretty big update to nuke ops and traitor uplink that I've failed to keep up with: largely a balance patch, but also has some new additions.

The above PRs explain it well, but I'll do a rough recap here:

Changes/Balance
- Camera bug no longer requires upgrading; it starts off fully upgraded
- Syndicate toolbox starts off with combat gloves as opposed to red insulated gloves
- Syndicate dufflebags and the syndicate  toolbox are now silent when you put items in them
- Reduces the cost of a lot of uplink items: Rad laser, Camera bug, dragon's breath rounds, chameleon suit, syndicate ID card, voice changer, syndi galoshes, syndi mag boots, space suit, and syndi-cigs.
- Prices increased on some uplink items: Pizza bomb and syndicate galoshes (for nuke ops ONLY on the galoshes).
- Syndicate hardsuits have THICKMATERIAL regardless of if they're in combat or travel mode
- Bulldogs start off with slugs as opposed to buckshot

Additions
- Adds in syndicate ammo  dufflebag: basically has a whole bunch of ammo in it for a bulldog--price efficient and a nice storage addition space
- Can purchase bulldog slugs on the uplink now
- Adds in bioterror syringes: they silence and knock you out, but don't do much damage: useful for stealth takeouts: for use in syringe guns
- Adds in military belts to the uplink (this is for REGULAR traitors, not nukies)
- Adds in surgery bag to the uplink. Basically a complete surgery kit+muzzle+straight jacket in a dufflebag.

### CODE REVIEW

File: code/game/objects/items/devices/camera_bug.dm [hunk 1]
```diff
@@ -88,26 +71,18 @@
 	return 1
 
 /obj/item/device/camera_bug/proc/get_cameras()
-	if(bugtype != last_bugtype || ( (bugtype in list(UNIVERSAL_BUG,NETWORK_BUG,ADMIN_BUG)) && world.time > (last_net_update + 100)))
+	if( world.time > (last_net_update + 100))
```
Comment (tigercat2000) [inline code/game/objects/items/devices/camera_bug.dm]: `if( world`
That pointless space, though.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -231,19 +231,65 @@
...
 	name = "suspicious duffelbag"
 	desc = "A black and red duffelbag with a patch depicting shotgun shells sewn onto it."
 	icon_state = "duffel-syndiammo"
 	item_state = "duffel-syndiammo"
-	slowdown = 0
+
+/obj/item/weapon/storage/backpack/duffel/syndie/ammo/loaded
+	desc = "A large dufflebag, packed to the brim with Bulldog shotgun ammo."
+
+	New()
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/backpack.dm]: Relative pathing?

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 3]
```diff
@@ -231,19 +231,65 @@
...
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g/buckshot(src)
+		new /obj/item/ammo_box/magazine/m12g/stun(src)
+		new /obj/item/ammo_box/magazine/m12g/dragon(src)
+		return
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/backpack.dm]: Pointless returns?

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 4]
```diff
@@ -231,19 +231,65 @@
...
+
+/obj/item/weapon/storage/backpack/duffel/syndie/surgery
+	name = "surgery dufflebag"
+	desc = "A suspicious looking dufflebag for holding surgery tools."
+	icon_state = "duffel-syndimed"
+	item_state = "duffle-syndimed"
+	storage_slots = 12
+
+	New()
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/backpack.dm]: The exact same thing on the next syndie bag?

Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/backpack.dm]: Didn't look at it close enough; I had just worked with syringe code (which is relatively pathed)--I didn't look close enough and thought this was too.

**Pull request merged**